package.json

```diff
@@ -44,21 +47,22 @@
   },
   "devDependencies": {
     "aws-sdk": "2.352.0",
-    "axios": "0.18.0",
     "babel-cli": "6.26.0",
     "babel-core": "6.26.3",
     "babel-preset-env": "1.7.0",
     "eslint": "5.9.0",
     "eslint-config-airbnb-base": "13.1.0",
     "eslint-plugin-import": "2.14",
     "jest": "23.6.0",
-    "lodash.filter": "4.6.0",
-    "uuid": "3.3.2"
+    "serverless": "1.33.2"
   },
   "peerDependencies": {
-    "aws-sdk": ">= 2.x.x",
-    "axios": ">= 0.x.x",
-    "lodash.filter": ">= 4.x.x",
-    "moment": ">= 2.x.x"
-  }
+    "aws-sdk": ">= 2.x.x"
+  },
+  "files": [
```

Comment:
List of files to publish to npm I assume?

Reply:
yes whitelist files to publish or you can also add a .npmignore if you want to blacklist :)
More easy in that way, i found @Vadorequest